### ISSUE TITLE
Remove dependency on TQDM, replace progress logging

### ISSUE THREAD

**Issue by NathanLovato** [GDquest/BPSRender#5]
We use a library to have a progress bar, tqdm. It makes it hard to parse the output of the renderer and adds an unecessary dependency. This would fix #4

**Comment by cgvirus** [comment]
Hi Nathan, will it be solved in future? I am planning to implement a simple Blender export mechanism to run bpsrender without CLI experience. Let me know Thanks.

**Comment by cgvirus** [comment]
Here are some nice boilers if you want to implement.
https://stackoverflow.com/questions/3160699/python-progress-bar

**Comment by NathanLovato** [comment]
We already have had some code and plan for that in Power Sequencer. See https://github.com/GDQuest/blender-power-sequencer/blob/master/operators/utils/info_progress_bar.py

It could be done in the future, but this is really low on my list of priorities at the moment. Contributions would be welcome!
